components/chiplist/overview.md

```diff
@@ -0,0 +1,99 @@
+
+1. Add the `TelerikChipList` tag
+1. Set the `Data` parameter to a collection of models that will be rendered as chips.
+1. Set the `TextField` parameter to point to the corresponding name in the model that holds the text that will be rendered in the individual chip. Read the [Data Binding article]({%slug chiplist-bound%}) for more information on other binding options.
+
+>caption Basic ChipList for Blazor
+
+````CSHTML
+<TelerikChipList Data="@ChipListSource"
+                 TextField="@(nameof(ChipModel.ChipText))"
+                 IconField="@(nameof(ChipModel.ChipIcon))">
+</TelerikChipList>
+
+@code {
+    private List<ChipModel> ChipListSource { get; set; } = new List<ChipModel>()
+    {
+        new ChipModel()
+        {
+            ChipText = "Audio",
+            ChipIcon = FontIcon.FileAudio
+        },
+        new ChipModel()
+        {
+            ChipText = "Video",
+            ChipIcon = FontIcon.FileVideo
+        }
+    };
+
+    public class ChipModel
+    {
+        public string ChipText { get; set; }
+        public FontIcon? ChipIcon { get; set; }
+    }
+}
+````
+
+## Data Binding
+
+The Blazor ChipList requires a data source so that it can display items to the user. To provide a data source, use the `Data` property. [Read more about the Blazor ChipList data binding]({%slug chiplist-bound%}).
+
+## Selection
+
+The Blazor ChipList supports single and multiple selection of chips. [Read more about the Blazor ChipList selection...]({%slug chiplist-selection%})  
+
+## Templates 
+
+You can use the functionality of the built-in templates and customize the default rendering of the component. [Read more about the Blazor ChipList templates...]({%slug chiplist-templates%})
+
+## Events
+
+You can use the built-in events of the Blazor ChipList to react to chip selection and removal. [Read more about the Blazor ChipList events...]({%slug chiplist-events%})
+
+## Appearance
+
+You can customize the appearance of the Blazor ChipList via a variety of built-in customization options. [Read more about the ChipList appearance settings...]({%slug chiplist-settings%})
+
+## ChipList Parameters
+
+The table below lists the ChipList parameters. Also check the [ChipList API Reference](/blazor-ui/api/Telerik.Blazor.Components.TelerikChipList) for all parameters, methods and events.
+
+@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
+
+| Parameter | Type and Default&nbsp;Value | Description |
+| --- | --- | --- |
+| `AriaLabel` | `string` | Maps to the `aria-label` attribute. Use  this parameter if the text that labels the component is not visible. |
+| `AriaLabelledBy` | `string` | Maps to the `area-labelledby` attribute. Use this parameter to reference another element to define its accessible name. |
+| `Class` | `string` | An additional CSS class for the `<div class="k-chip-list">` element. Use it to [customize the component styles and override the theme]({%slug themes-override%}). |
```

Comment:
I suggest moving the `Class` parameter and the appearance section to a dedicated "Styling and appearance" subsection. Ref: https://docs.telerik.com/blazor-ui/components/autocomplete/overview#styling-and-appearance

Reply:
I am a bit against that. The class can be used for more things than simply styling the component. The Aria* is accessibility related, did I miss something in the comment?